docs/reference/components/sinks/aws_cloudwatch_logs.cue

```diff
@@ -5,7 +5,7 @@ components: sinks: aws_cloudwatch_logs: components._aws & {
 
 	classes: {
 		commonly_used: true
-		delivery:      "at_least_once"
+		delivery:      "best_effort"
```

Comment:
🤔 There is some nuance here that it is only `at_least_once` when disk buffers are used, but I think this and many of the below components are still `at_least_once`, no? I think only protocols that don't support acknowledgement, like UDP, would be `best_effort`. This is mentioned in the above guarantees.md:

> In order to achieve at least once delivery between restarts your sink must be configured to use `disk` based buffers:

End-to-end acknowledgements change this, but at a topology level rather than a component level, I think.

Reply:
Before this change, the `at_least_once` indicated simply that we would retry until the events were accepted or explicitly rejected. Now, it has been overloaded to also indicate that it participates in end-to-end acknowledgements. I am open to a better solution to this, as I don't particularly like this overloading, but see also this: https://github.com/timberio/vector/pull/7436#discussion_r634764018

Reply:
Agreed, I think we'll need to come up with some better terms and definitions as part of this. We can delay thinking about it too much until we pick up the work to officially release end-to-end acknowledgements though.